modules/ebi-metagenomics/eggnogmapper/meta.yml

```diff
@@ -27,6 +27,11 @@ input:
       description: Database of sequences in FASTA format
       pattern: "*.{fasta,fa,fasta.gz,fa.gz}"
```

Comment:
If you are removing the support for compressed files, you'll have to remove it from here as well